src/utils/client.js

```diff
@@ -0,0 +1,128 @@
+import ApolloClient, { gql } from 'apollo-boost';
+
+export const REACT_APP_API_URL =
+  window.env && window.env.REACT_APP_API_URL
+    ? window.env.REACT_APP_API_URL
+    : process.env.REACT_APP_API_URL;
+
+const client = new ApolloClient({
+  uri: process.env.REACT_APP_API_URL
+});
+
+const getAllNews = () =>
+  client.query({
+    query: gql`
```

Comment:
move queries to saga from apollo client. apollo client should contain only the main query  for query and mutations, not all of them

Reply:
Resolved!